web/italianTranscription.html

```diff
@@ -380,12 +380,12 @@ <h4 id="editBtns" title="Edit available XML Tags and Special Characters" class="
             
             <div id="startSplit" class="split">
                 <div style="text-align: center;" class="fullScreenTrans"><i class="fa fa-share fa-flip-vertical"></i></div>
-                <iframe class="loadingIframe" data_src="new-paleography?response_type=embed"></iframe>
+                <iframe class="loadingIframe" data_src="TODO?response_type=embed"></iframe>
```

Comment:
Ah shoot I forgot I had been tempted to do both of these at the  same time, and so real early on I actually made some changes on this page.  They are correct at least...do you want me to undo these or keep them?